content/docs/guides/branching-github-actions.md

```diff
@@ -286,6 +287,137 @@ outputs:
 - `host_with_pooler`: The branch host with pooling after the reset.
 - `password`: The password for connecting to the branch database after the reset.
 
+## Schema Diff action
+
+This action performs a database schema diff on specified Neon branches for each pull request and writes a **Neon Schema Diff summary** comment to the pull request in GitHub highlighting the schema differences.
+
+It supports workflows where schema changes are made on a development branch, and pull requests are created for review before merging the changes back into the main branch. By including the schema diff as a comment in the pull request, reviewers can easily assess the changes directly within the pull request.
+
+<Admonition type="tip" title="GitHub Actions Marketplace">
+You can find this action on the **GitHub Actions Marketplace**: [Neon Schema Diff GitHub Action](https://github.com/marketplace/actions/neon-schema-diff-github-action).
+</Admonition>
+
+### Prerequisites
+
+Using the action requires adding a Neon API key to your GitHub Secrets. For information about obtaining an API key, see [Create an API key](/docs/manage/api-keys#create-an-api-key).
+
+1. In your GitHub repository, go to **Project settings** and locate **Secrets** at the bottom of the left sidebar.
+2. Click **Actions** > **New Repository Secret**.
+3. Name the secret `NEON_API_KEY` and paste your API key in the **Secret** field
+4. Click **Add Secret**.
+
+Alternatively, the **Neon GitHub Integration** can perform the API key setup for you. See [Neon GitHub integration](/docs/guides/neon-github-integration).
```

Comment:
Swap the integration to be the first one in the list

Reply:
Swapped, but rather than repeating this section for all four actions in this topic, I've added a separate section for it which is linked from the prereqs.